src/Bicep.Cli.E2eTests/src/examples/101/aks.ff/flawed.bicep

```diff
@@ -54,4 +54,3 @@ resource aks 'Microsoft.ContainerService/managedClusters@2020-09-01' = {
 }
 
 output controlPlaneFQDN string = aks.properties.fqdn
-
```

Comment:
My preference would be not to trim trailing newlines

Reply:
By updating .editorconfig, you have also option to introduce rules that later on as also consumed be IDE editors and cli's. Whitespace is only one of basic ones fixing readability, In these cases to make clear separation it is good practice to make separate commits doing fixing of current source. Applaying ruels can be done by vs code, or pre-commit etc. So before any commit, rules can be forced/appealed.

Reply:
The .editorconfig file does not provide any rules for .bicep files, so this file should not have been changed.